packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+/** Props for ProgressBar */
+interface ProgressBarProps {
+  animate: boolean /** apply animation to progressBar */;
+  decimalPoints: number /** Rounds off the value to this number */;
+  height: string /** resizes the height of the progressBar */;
+  min: number /** set lower bound for the progressBar range */;
+  max: number /** set upper bound for the progressBar range */;
+  value: number /** Represents the progress bar value */;
+  lineColor: string /** set line colors */;
+  lineColorThresholds?: { [key: string]: number } /** set linecolor threshold */;
+  progressClass: string /** sets the class for progressBar */;
+  showLabel: boolean /** set label on progressBar */;
+  stripped: boolean /** set strips in progressBar */;
+}
+
+/** default props for ProgressBar */
+const defaultProgressBarProps = {
+  animate: false,
+  decimalPoints: 0,
+  height: '10px',
+  lineColor: '#0000FF',
+  lineColorThresholds: undefined,
+  max: 100,
+  min: 0,
+  progressClass: 'progress-bar-striped progress-bar-animated',
+  showLabel: false,
+  stripped: false,
+  value: 0
+};
+
+/** Displays configurable progress bar
+ * lineColor prop when you don't require different line colors
+ * lineColorThresholds when you have to match lineColors to certain value thresholds.
+ * LineColorThresholds will take precedence over lineColor if both are provided
```

Comment:
I think we should remove line 34 - 36.

Reply:
1. It's fine by me to have the name as `cssClass`. It makes it more elaborate.
2. I guess I can add examples of cssClass to the comment? I am not entirely sure
3. That makes sense
4. It's nested because there is a `multiple bars` component that needs to be added later on(need to figure out how to do that). That is the reason we have `ProgressIndicators` as the name of the package and `Progress bar`  as one of the component's directory.